### PR TITLE
Removes useless commented-out what-if code from the 1970s

### PR DESCRIPTION
**Pull request by vuonojenmustaturska**
If someone is making a /datum/New(), I'm sure they'll know where they need to place a call to parent in order for it to ever be run. If they don't, we are better off if it never runs.

There were other examples of uselessness like these, mostly debug verbs, but I kept them in case someone wants to, for example, make experimentor useful and not a waste of processing and mapspace again.

[why]: 
Removing clutter

### CODE REVIEW

**Comment by vuonojenmustaturska** [comment]
Also removed Errorage's tales of salt and violence against game mechanics that slighted him, another commented out code block from when Reagan was president, and a suggestion to seek approval from Errorage to make changes to a following define block

**Review by Cyberboss — DISMISSED** [review]

File: code/_globalvars/lists/mapping.dm [hunk 1]
```diff
@@ -9,9 +9,6 @@ GLOBAL_LIST_INIT(diagonals, list(NORTHEAST, NORTHWEST, SOUTHEAST, SOUTHWEST))
 
 //This list contains the z-level numbers which can be accessed via space travel and the percentile chances to get there.
 //(Exceptions: extended, sandbox and nuke) -Errorage
-//Was list("3" = 30, "4" = 70).
-//Spacing should be a reliable method of getting rid of a body -- Urist.
-//Go away Urist, I'm restoring this to the longer list. ~Errorage
 GLOBAL_LIST_INIT(accessable_z_levels, list(1,3,4,5,6,7)) //Keep this to six maps, repeating z-levels is ok if needed
```
Comment (ChangelingRain) [inline code/_globalvars/lists/mapping.dm]: defines, why aren't these defines

**Review by Cyberboss — DISMISSED** [review]

**Review by Cyberboss — DISMISSED** [review]

**Review by Cyberboss — APPROVED** [review]
Reee

**Comment by YakumoChen** [comment]
But why remove any of this? It's funny

**Comment by Bawhoppen** [comment]
These are antiques. Leave them.

**Review by jughu — APPROVED** [review]

**Comment by Bawhoppen** [comment]
Boo

**Pull request merged**